data/brands/shop/clothes.json

```diff
@@ -3200,6 +3200,17 @@
         "shop": "clothes"
       }
     },
+    {
+      "displayName": "H&M Kids",
+      "locationSet": {"include": ["gb"]},
+      "tags": {
+        "brand": "H&M Kids",
+        "brand:wikidata": "Q188326",
```

Comment:
I forget, we don't need a separate QID do we?

Reply:
Some entries are using country/region's branch company's QID, and on Wikidata, those QID have a same parent brand.
Another and more often time, we use the same QID for every country/region's entries. 
We current don't have guide rule about in which time we should use QID in which hierarchy, global one or regional one. So I think Q188326 for H&M Kids is OK.

Reply:
Yes, this is ok, we don't strictly need a separate QID for each sub-brand.

For comparison, the Gap sub-brands all use `brand=Gap` and `brand:wikidata=Q420822`
https://nsi.guide/index.html?t=brands&k=shop&v=clothes&tt=Gap

So I think here it would be ok to use `brand=H&M` and `brand:wikidata=Q188326`.
(but `brand=H&M Kids` also seems acceptable to me)

Reply:
Yeah that makes sense, I hadn't really considered the actual brand tag, now updated.